### PR TITLE
Decode html chars in dashboard greeting translation

### PR DESCRIPTION
**Pull request by julien-nc**
refs #26113 

Not an ideal solution but it works.
This could be done in many other places.

### CODE REVIEW

**Comment by skjnldsv** [comment]
> Not an ideal solution but it works.
> This could be done in many other places.

What about putting this into @nextcloud/l10n then?

File: apps/dashboard/src/App.vue [hunk 1]
```diff
@@ -396,6 +396,11 @@ export default {
 				document.body.classList.remove('dashboard--scrolled')
 			}
 		},
+		htmlDecode(s) {
+			return s.replace(/&#(\d+);/g, (match, code) => {
```
Comment (skjnldsv) [inline apps/dashboard/src/App.vue:400]: Don't we have a lib for it?
I would prefer not to have random pieces of code here and there?

https://www.npmjs.com/package/html-entities
https://github.com/nextcloud/server/blob/6b4cfc580588b459fb3ca6053f645633241fdd7a/package.json#L55

html-entities seems a bit big, but we have to see what real size is being added to the webpack bundle?

**Comment by eneiluj** [comment]
@skjnldsv Do you mean adding and exporting a decode function in @nextcloud/l10n and letting developers use it when they want
**or** calling it for every result of translate and translatePlural?

**Comment by skjnldsv** [comment]
> Do you mean adding and exporting a decode function in @nextcloud/l10n and letting developers use it when they want
> **or** calling it for every result of translate and translatePlural?

No I mean fixing the t and n function to automatically htmldecode?

**Pull request closed without merging**